src/wireviz/DataClasses.py

```diff
@@ -18,6 +18,10 @@ class Connector:
     type: Optional[str] = None
     subtype: Optional[str] = None
     pincount: Optional[int] = None
+    image: Optional[str] = None
+    image_scale: Optional[str] = "FALSE"
+    image_size: List[Any] = field(default_factory=list)
```

Comment:
Does it make sense to have a list here, instead of separate `image_width` and `image_height` attributes of type `int`? That removes the guessing of what order the list is (width or height first).

Reply:
I can do that if that's preferred. I just read somewhere that it's more Pythonic to specify a width x height size as a tuple, but I'm not sure how to specify a tuple in the YAML input.

Reply:
AFAIK there are no tuples in YAML. And expecially having the`fixed` parameter in there (which might be a bit cryptic for a regular user) is not very pythonic IMHO, so having an easy to read syntax [as suggested](https://github.com/formatc1702/WireViz/pull/153#discussion_r470489101) is probably the user-friendliest compromise.

The whole width x height tuple sounds like something out of Numpy, or OpenCV maybe, but leaving out the guesswork for the user is preferrable IMO.